### ISSUE TITLE
User templates not Showing Up in the Insert track from template Submenu

### ISSUE THREAD

**Issue by MatejGolian** [jcsteh/osara#118]
Track templates stored in the default track template directory don't show up in the
"Insert track from template" submenu when you press CTRL+Apps. I see an item "(track template list)" right where the track templates should be.
The only way of getting the complete context menu at the moment is manually right-clicking a track.

**Comment by jcsteh** [comment]
This is due to the rather hacky way OSARA currently accesses these context menus. Unfortunately, as far as I've been able to discover, there isn't a proper way to do this which doesn't suffer from this bug.

I contacted Cockos about this (and issues with other context menus) on 14 June, but haven't had any response. I imagine they're super busy with REAPER 5.50, so I'll wait until after that is released before following up.
